CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+We are glad that you are interested in contributing to the Plasma Engine! We are happy to accept contributions, including:
+- bug reports
+- bug fixes
+- feature requests
+- new features
+- art assets
+- and much more!
+
+Please take a moment to read these guidelines to ensure your contributions are accepted.
+
+## Project Goals
+The goal of the H-uru Plasma project is to improve on the open source release of the CyanWorlds.com Engine used in Myst Online: Uru Live. In the interest of doing so, we have identified these project goals:
+- improving cross-platform and compiler compatibility
+- removing dependencies on nonfree libraries
+- fixing technical debt by utilizing modern language features in both C++ and Python
+- improving accessibility for fan Age creation and testing
+- improving testing coverage by adding unit tests and continuous integration
+- improving the player experience
+- avoiding nontrivial breaks in compatibility with the official Myst Online: Uru Live (again) game run by Cyan Worlds, Inc.
+
+Further, we have identified these non-goals:
+- maintaining a game engine to be used outside of the context of Myst Online: Uru Live
+- supporting legacy platforms, such as Windows 2000, Windows XP, Playstation 2, and XBox
+- supporting exploit-based gameplay
+
+Information about specific tasks can be found on the [project roadmap](https://github.com/H-uru/Plasma/wiki/Roadmap).
+
+## Getting Involved
+Real-time discussion with team members and stakeholders is an excellent way to begin contributing. We welcome feedback and discussion of proposed changes. Active maintainers can be found on the Guild of Writers IRC channel:
```

Comment:
"stakeholders" feels too specific and formal.  Perhaps "other contributors" is sufficient?

Reply:
I blame my education background where "stakeholders" is a common term 😛

Reply:
I went back and forth on this one a few times.  It's concise, which is good, but I feel that it carries ownership connotations in everyday speech that isn't appropriate here.